.github/workflows/daily-rubygems.yml

```diff
@@ -16,14 +16,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-24.04 ]
-        ruby: [ ruby-head, truffleruby-head ]
+        os: [ubuntu-24.04]
+        ruby: [ruby-head, truffleruby-head]
```

Comment:
I assume zizmor does not do any styling and just cares about security, should we do some YAML style linting too to keep a consistent workflow style and avoid this kind of style only changes?

Reply:
> I assume zizmor does not do any styling and just cares about security, should we do some YAML style linting too to keep a consistent workflow style and avoid this kind of style only changes?

Correct! You could use `yamllint` or similar for this, but that's not something I have strong opinions about (as the `zizmor` author).